start_project_production.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/bash
 
-poetry install
-poetry run gunicorn --bind 0.0.0.0:8000 "app:create_app()"
\ No newline at end of file
+poetry run gunicorn --bind 0.0.0.0:$PORT "app:create_app()"
```

Comment:
This now **requires** a PORT env var to be set. So your README/docker-compose.prod.yml are out of date.

Best and easiest fix is to set a default value for PORT. Either set e.g. `ENV PORT=8000` in the Dockerfile or you can use `0.0.0.0:${PORT:-8000}` here.